### PR TITLE
(maint) Clarify supported PE and Bolt versions

### PR DESCRIPTION
**Pull request by eimlav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,10 +17,12 @@ This module provides the resource task. This task allows you to inspect the valu
 ## Requirements
 This module is compatible with Puppet Enterprise and Puppet Bolt.
 
-* To run tasks with Puppet Enterprise, PE 2017.3 or later must be installed on the machine from which you are running task commands. Machines receiving task requests must be Puppet agents.
+* To run tasks with Puppet Enterprise, PE 2018.1 or later must be installed on the machine from which you are running task commands. Machines receiving task requests must be Puppet agents.
 
 * To run tasks with Puppet Bolt, Bolt 1.0 or later must be installed on the machine from which you are running task commands. Machines receiving task requests must have SSH or WinRM services enabled.
 
+The module may still be compatible with older verions of Puppet Enterprise and Puppet Bolt however we do not support these and their use is not advised.
```
Comment (clairecadman) [inline README.md]: Typo: "versions"

Slight reword:

The module may still be compatible with older versions of Puppet Enterprise and Puppet Bolt, however we do not support these and advice against it.

Why "may still be compatible", do we not know for sure whether it is or not?

Comment (eimlav) [reply README.md]: We only support Puppet PE 2018.x or later and Bolt 1.x or later but the module should still work with some older versions of both of these such as PE 2017.3 and Bolt 0.23. Do you think we should just outright remove this section possibly?

Comment (clairecadman) [reply README.md]: Yeah maybe just remove all together? Especially as it says above that 2018.1 or later *must* be installed. I think it's better not to mention it if it's not advised. Maybe double check with someone else though.

Comment (eimlav) [reply README.md]: We decided to just remove that section. Thanks for the feedback!

**Pull request merged**